contracts/Hangman.sol

```diff
@@ -35,12 +47,21 @@ contract Hangman {
         }
     }
 
-    function makeWordGuess() external {
+    function makeWordGuess(bytes calldata _string) external {
```

Comment:
`require` the incoming string to be the same length as the solution string.

Reply:
i'm adding the changes in a new commit